packages/utils/src/fhir-deduplication/check-references.ts

```diff
@@ -0,0 +1,51 @@
+import * as dotenv from "dotenv";
+dotenv.config();
+// keep that ^ on top
+import { BundleEntry, Resource } from "@medplum/fhirtypes";
+import { out } from "@metriport/core/util/log";
+import fs from "fs";
+import { elapsedTimeAsStr } from "../shared/duration";
+import { initRunsFolder } from "../shared/folder";
+import { getMissingReferences } from "./report/validate-references";
+
+/**
+ * Utility to validate references on a FHIR bundle.
+ *
+ * Usage:
+ * - set the path to the file on `filePath`
+ * - run `ts-node src/fhir-deduplication/validate-references.ts`
+ */
+const filePath = "";
+
+async function main() {
+  const startedAt = Date.now();
+  initRunsFolder();
+  const { log } = out(``);
+
+  const bundleFile = fs.readFileSync(filePath, "utf8");
+
+  const resources: Resource[] =
+    JSON.parse(bundleFile).entry?.map((entry: BundleEntry) => entry.resource) ?? [];
+
+  log(`Validating references on bundle (${resources.length} resources)...`);
+  const res = getMissingReferences(resources);
```

Comment:
we already have a function for this in core called `getReferencesFromResources` that returns missing references. Would it be better to use that for this util script?

Reply:
Good catch! I didn't remember about that one.

I'll compare the output of the two to make sure we're not missing anything, and leave a single one on this PR.